docs/guide/community-spotlight-interviewing-guide.rst

```diff
@@ -0,0 +1,163 @@
+
+Editing the transcript into the first draft
+-------------------------------------------
+
+Each interview should contain the career summary of the interview subject along with the advice they would give to new writers. Those answers together will total roughly 400-700 words and will generally be about half of the first draft. The content of the other half of the document is up to the interviewers discretion.
+
+When considering what to include in the other half, the interviewer should consider the audience and what aspects of the interviewee's experience provides the most utility for an inexperienced tech writer. For example, a conversation about document design may be interesting, but a conversation about working as a freelance writer would be more relevant, and more broadly relevant to the intended audience. Preference should also be given to topics that haven't been thoroughly covered by other write ups.
+
+Preparing the draft for a pull request
+--------------------------------------
+
+Before committing the draft publicly and submitting the pull request to the WTD repo, the draft should be sent its respective interviewee so that they may clarify answers or remove any personal information. Once the interviewee has accepted the draft as being appropriate for publishing, the document can be committed publicly to your own personal fork of the main WTD website repo. Each interview should go in it's own branch.
+
+Formatting and other extra considerations
+-----------------------------------------
+
+Every interview is written in RST. For an example of what the RST formatting should look like, you can reference `this example`_ or any of the other documents in the community-spotlight directory of the repository.
+
+Companies and well known technologies or tools should link out to relevant web pages. Company links should redirect to their careers page, if available. Technology and tools links should redirect to the standard resource for learning that technology or the WTD page if it exists.
+
+General questions resource
+==========================
+
+**Education**
+
+    What formal education do you have?
+        Would you consider pursuing more formal education?
+
+Where have you learned the skills that you use for your work?
+
+**Work**
+    
+    What is the story of your career so far?
+
+    What other jobs have you had?
+        How has that experience influenced your work now?    
+
+    How long did it take between learning you wanted to become a tech writer and getting 
```

Comment:
Some weird formatting here. We should probably just make these actual bullet points (`*`) indented properly, so they render as a list.